app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,17 @@
+from abc import ABC
+from app.players.player import Player
+
+
+class Dwarf(Player, ABC):

```

Comment:
Since Player is already inheriting from ABC, there's no need for Dwarf to inherit from ABC directly.